pyproject.toml

```diff
@@ -38,6 +38,7 @@ pytz = ">=2019.1"
 pandas = "~1.3"
 requests = ">=2.26.0"
 tenacity = ">=5.1.0"
+twilio = ">=7.14.0"
```

Comment:
I'm wondering if we want to set this as optional and install this as an extra `pip install camply[twilio]` since it only applies to one notification provider. What do you think?

Reply:
Yes, I think that makes a lot of sense. Thanks for the pointer on how to mark it optional.